iOS/CodePushPackage.m

```diff
@@ -203,7 +203,9 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
            doneCallback:(void (^)())doneCallback
            failCallback:(void (^)(NSError *err))failCallback
 {
-    NSString *newPackageFolderPath = [self getPackageFolderPath:updatePackage[@"packageHash"]];
+    NSString *newPackageHash = updatePackage[@"packageHash"];
```

Comment:
While we're making some refactoring changes, should we replace the use of the word "Package" for "Update" (e.g. `newUpdateHash`, `newUpdateDirectoryPath`).